engines/wintermute/base/gfx/opengl/meshx.cpp

```diff
@@ -213,6 +213,33 @@ bool MeshX::update(FrameNode *parentFrame) {
 				}
 			}
 		}
+
+		for (uint i = 0; i < skinWeightsList.size(); ++i) {
```

Comment:
These loops need a bit more comments to make it easier to keep track of what we're doing, why.